packages/components/Checkbox/src/Checkbox.tsx

```diff
@@ -55,23 +55,14 @@ export const Checkbox = compose<ICheckboxType>({
       [toggleChecked, userProps, state, pressable.props]
     );
 
-    let accessibilityStates: string[] = [];
-    if (state.disabled) {
-      accessibilityStates = ['disabled'];
-    } else if (state.checked) {
-      accessibilityStates = ['checked'];
-    } else {
-      accessibilityStates = ['unchecked'];
-    }
-
     const slotProps = mergeSettings<ICheckboxSlotProps>(styleProps, {
       root: {
         rest,
         ref: buttonRef,
         ...pressable.props,
         accessibilityRole: 'checkbox',
         accessibilityLabel: ariaLabel || label,
-        accessibilityStates: accessibilityStates,
+        accessibilityState: {disabled: state.disabled, checked: state.checked },
         accessibilityActions: [{ name: 'Toggle', label: checkboxSelectActionLabel }],
         onAccessibilityAction: onAccessibilityAction,
         onKeyUp: onKeyUpSpace
```

Comment:
I noticed trailing commas were added to Button and RadioGroup when I saved those files but nothing was effected in Checkbox. Any idea why? I thought @JasonVMo added something to make trailing commas standard for the repo.

Reply:
Did you use VS Code?  That should come from the eslint config and the eslint plugin in VS Code.